src/evidently/suite/base_suite.py

```diff
@@ -253,9 +261,9 @@ def save_html(self, filename: Union[str, IO], mode: Union[str, SaveMode] = SaveM
                 dashboard_info=dashboard_info,
                 additional_graphs=graphs,
             )
-            render = self._render(determine_template("inline"), template_params)
+            render = self._render(file_html_template, template_params)
```

Comment:
Pay attention that we are using `file_html_template` now here with `<meta charset="UTF-8">` tag inside